modules/openid_auth/app/controllers/openid_auth/v0/okta_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_dependency 'openid_auth/application_controller'
+
+module OpenidAuth
+  module V0
+    class OktaController < ApplicationController
+      skip_before_action :authenticate
+
+      def metadata
+        meta = OneLogin::RubySaml::Metadata.new
+        render xml: meta.generate(saml_settings), content_type: 'application/xml'
+      end
+
+      def parse_user_attributes(json_request)
+        okta_attributes = json_request.dig(:data, :assertion, :claims)
+        user_attributes = Hash.new
+        okta_attributes.each do |k, v|
+          user_attributes[k.to_sym] = v[:attributeValues][0][:value]
+        end
+        user_attributes
+      end
+
+      def fetch_mvi_profile(user_attributes)    
+        user_identity = OpenidUserIdentity.new(
+          mhv_icn: user_attributes[:mhv_icn]
+        )
+
+        mvi_response = MVI::Service.new.find_profile(user_identity)
+
+        raise mvi_response.error if mvi_response.error # TODO: add error logging
+        mvi_response[:profile]
+      end
+
+      def okta_callback      
+        user_attributes = parse_user_attributes(params)
+        mvi_profile = fetch_mvi_profile(user_attributes)
+
+        render json: {
+          "commands": [
+            {
+              "type": "com.okta.assertion.patch",
+              "value": [
+                {
+                  "op": "replace",
+                  "path": "/claims/dslogon_edipi/attributeValues/0/value",
+                  "value": mvi_profile[:edipi]
```

Comment:
this and the added value are arbitrary. just wanted to verify things were getting modified by Okta correctly and then update to what we want to pass through